bin/gen_otel_config.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
```

Comment:
This kind of initialization code currently is done on [_setup.py file](https://github.com/resource-hub-dev/rhub-api/blob/master/src/rhub/api/_setup.py). Unless you think it makes no sense (specially in long term), I'd suggest to move what is relevant to the `_setup.py` file and make the call to `bm_hosts_to_monitor_list` directly, without the HTTP request.

Reply:
Right now it needs to be run in a separate form; the next step will be adding both OTEL and this startup code in a different image, so a new configuration is generated upon each restart.

Reply:
Removed as well

Reply:
This file is still present. I'm ok with this, just pointing out to check your intention...

Reply:
This will be moved into a different directory in the next merge (together with the other OTEL bits.)